internal/build/imgsrc/ensure_builder_test.go

```diff
@@ -275,16 +276,24 @@ func TestRestartBuilderMachine(t *testing.T) {
 			return nil
 		},
 		WaitFunc: func(ctx context.Context, machine *fly.Machine, state string, timeout time.Duration) (err error) {
+			if state == "started" || state == "stopped" {
+				waitedForStartOrStop = true
```

Comment:
I don't understand how this test is asserting the logic correctly since the initial state of `waitedForStartOrStop` is `true`.

Reply:
oh, yea i should set that to false by default 🤦🏾